src/containerapp/azext_containerapp/containerapp_decorator.py

```diff
@@ -553,7 +583,7 @@ def construct_containerapp_for_post_process(self, containerapp_def, r):
             registries_def["server"] = self.get_argument_registry_server()
             registries_def["identity"] = self.get_argument_registry_identity()
             safe_set(containerapp_def, "properties", "configuration", "registries", value=[registries_def])
-            return containerapp_def
+        return containerapp_def
```

Comment:
nit: stylistically this looks correct, but just want to ensure this change was intended

Reply:
This change was not intended.